packages/datadog-plugin-redis/src/index.js

```diff
@@ -68,8 +68,8 @@ class RedisPlugin extends Plugin {
   }
 
   configure (config) {
-    Plugin.prototype.configure.call(this, config)
-    return normalizeConfig(this.config)
+    normalizeConfig(config)
```

Comment:
This has no effect as it returns a new object that's never assigned to anything. This is why I mentioned that a test is clearly missing.